contrib/firmware-packager/firmware-packager

```diff
@@ -24,14 +24,14 @@ def cd(path):
 firmware_metainfo_template = """
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="firmware">
-  <id>{firmware_id}</id>
+  <id>org.freedesktop.fwupd.guid{firmware_id}</id>
```

Comment:
I don't really like this; the ID is the project that is responsible for the firmware, and in this case I think it's possibly dangerous to imply this is shipped by the LVFS or fwupd projects.

Reply:
I see what you mean. What about using developer-name. Like `com.(developer-name).guid(firmware-info)` and making developer-name mandatory?

I just don't think that the ID should be something that needs to be figured out by someone packaging this.

Reply:
Adjusted

Reply:
I don't really have a strong opinion on this but I am curious why you feel the user shouldn't pick the name for themselves. Is it because the form is somewhat complex and the goal of the script is to get up and running quickly?

Reply:
I'm hoping this script will be what we tell new IHVs and OEMs to use in their build processes to package "simple" firmware and it's a low level detail that most manufacturers shouldn't need to bother themselves with.

Reply:
OK I see I had assumed this was just a hack for users. In that case you probably want a whole lot more flexibility especially in naming